### PR TITLE
Quickstart add basic REST service

### PR DESCRIPTION
**Pull request by gberl002**
Also some extensive test revamping.

### CODE REVIEW

File: quickstart/docker/docker-compose.yml [hunk 1]
```diff
@@ -53,6 +54,7 @@ services:
       - ZITI_EDGE_CONTROLLER_RAWNAME="${ZITI_EDGE_CONTROLLER_RAWNAME}"
       - ZITI_EDGE_ROUTER_RAWNAME=ziti-edge-router
       - ZITI_EDGE_ROUTER_ROLES=public
+      - ZITI_EDGE_ROUTER_IDENTITY_ROLES=ziti.rest.host
```
Comment (dovholuknf) [inline quickstart/docker/docker-compose.yml:57]: i like the ability to add roles this way, but doesn't this belong in the .env file?

File: quickstart/test/quickstart_test.go [hunk 2]
```diff
@@ -518,7 +293,23 @@ func getIdentityByName(client *rest_management_api_client.ZitiEdgeManagement, na
...
+	return resp.GetPayload().Data[0]
+}
+
+func getRouterByName(client *rest_management_api_client.ZitiEdgeManagement, name string) *rest_model.RouterDetail {
+	filter := "name=\"" + name + "\""
+	params := &router.ListRoutersParams{
+		Filter:  &filter,
+		Context: context.Background(),
+	}
+	params.SetTimeout(30 * time.Second)
```
Comment (dovholuknf) [inline quickstart/test/quickstart_test.go:309]: that's a long time. does it need to be that long?

File: quickstart/test/dockerExpressInstall.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+Prerequisites:
+  * For some tests, you must have "ziti-edge-controller" added to your hosts file
+'
+
+# Build the docker image
+docker build ../docker/image -f ../docker/image/TestDockerfile -t openziti/quickstart:test
+docker rmi "$(docker images -f "dangling=true" -q)"
+
+# Start up the docker container, this will run expressInstall
+docker run --name quickstart-test -p 1280:1280 -p 3022:3022 -e ZITI_EDGE_CONTROLLER_RAWNAME=ziti-edge-controller -e ZITI_CONTROLLER_RAWNAME=ziti-edge-controller -e ZITI_EDGE_ROUTER_RAWNAME=ziti-edge-router openziti/quickstart:test
```
Comment (dovholuknf) [inline quickstart/test/dockerExpressInstall.sh:20]: this seems very helpful, i like it!

**Comment by gberl002** [comment]
Git got messed up, putting up a new PR

**Pull request closed without merging**